components/button.css

```diff
@@ -0,0 +1,115 @@
+@import "../base.css";
+@import "../utility.css";
+.btn {
+  display: inline-block;
+  cursor: pointer;
+  padding: 8px 1rem;
+  margin: 1rem 8px;
+  border-radius: 5px;
+  border: 1px solid transparent;
+  text-align: center;
+  vertical-align: middle;
+}
+.btn-primary:hover,
+.btn-secondary:hover,
+.btn-success:hover,
+.btn-danger:hover,
+.btn-warning:hover {
+  border: 2px solid var(--grey-border-color);
+  box-shadow: 5px 5px var(--grey-border-color);
+}
+
+/* Default buttons */
+.btn-primary {
+  background-color: var(--component-primary-color);
+}
+
+.btn-secondary {
+  background-color: var(--component-secondary-color);
+}
+.btn-success {
+  background-color: var(--component-success-color);
+}
+.btn-danger {
+  background-color: var(--component-error-color);
+}
+
+.btn-warning {
+  background-color: var(--component-warning-color);
+}
+/* Outline buttons */
+.btn-outline-primary {
+  color: #007bff;
+  background-color: transparent;
+  background-image: none;
+  border-color: #007bff;
+}
+
+.btn-outline-primary:hover {
+  color: var(--text-white-color);
+  background-color: #007bff;
+  background-image: none;
+  border-color: #007bff;
+}
```

Comment:
Already had too many variables for colors, and this colors were specific to outline buttons.so, used it directly instead of variables .If I have to reuse at any point, will change it to variables